src/Query/User/PackageQuery/DbalPackageQuery.php

```diff
@@ -29,10 +30,37 @@ public function findAll(string $organizationId, int $limit = 20, int $offset = 0
         return array_map(function (array $data): Package {
             return $this->hydratePackage($data);
         }, $this->connection->fetchAll(
-            'SELECT id, organization_id, type, repository_url, name, latest_released_version, latest_release_date, description, last_sync_at, last_sync_error, webhook_created_at
-            FROM "organization_package"
-            WHERE organization_id = :organization_id
-            ORDER BY name ASC
+            'SELECT
+                p.id,
+                p.organization_id,
+                p.type,
+                p.repository_url,
+                p.name,
+                p.latest_released_version,
+                p.latest_release_date,
+                p.description,
+                p.last_sync_at,
+                p.last_sync_error,
+                p.webhook_created_at,
+                (
+                    SELECT date FROM organization_package_scan_result s
+                    WHERE s.package_id = p.id
+                    ORDER BY date DESC LIMIT 1
+                ) scan_result_date,
+                (
+                    SELECT status FROM organization_package_scan_result s
+                    WHERE s.package_id = p.id
+                    ORDER BY date DESC LIMIT 1
+                ) scan_result_status,
+                (
+                    SELECT content FROM organization_package_scan_result s
+                    WHERE s.package_id = p.id
+                    ORDER BY date DESC LIMIT 1
+                ) scan_result_content
```

Comment:
we need to rewrite this query, fortunately, we have already agreed on a better solution